weave/trace/op_extensions/accumulator.py

```diff
@@ -82,11 +83,10 @@ def __iter__(self) -> "_IteratorWrapper":
 
     def __next__(self) -> Generator[None, None, V]:
         if not hasattr(self._iterator_or_ctx_manager, "__next__"):
-            raise TypeError(
-                f"Cannot call next on an iterator of type {type(self._iterator_or_ctx_manager)}"
-            )
+            self._iterator_or_ctx_manager = iter(self._iterator_or_ctx_manager)  # type: ignore
```

Comment:
Why are you putting a type ignore here?

Is this safe?  I think it assumes that whatever is yielded by the contextmanager is iter-able

Reply:
If I don't put type ignore here, its failing the mypy checks.

Reply:
The point of mypy is to check if it's type-safe, and the ignore here suggests that it's not.  Can you please remove the ignore and fix?

Reply:
We added this back then when the class was extended to support context manager for Anthropic, I pushed a hacky fix